torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3196,6 +3196,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_normal',
                                 dtypes=[torch.cfloat, torch.cdouble]),
                    )),
+    OpInfo('logaddexp',
+           dtypes=floating_types(),
+           sample_inputs_func=lambda op_info, device, dtype, requires_grad=False, **kwargs:
```

Comment:
haha; I'll allow it

Reply:
I felt it would be overkill to write a full function just for that.